tools/push_api_gh_pages.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -ev
+
+GH_REPO="@github.com/elopio/snapcraft.git"
```

Comment:
Can you push to a `gh-pages` branch on github.com:snapcore/snapcraft.git instead?